dask/dataframe/io/tests/test_parquet.py

```diff
@@ -4740,6 +4740,18 @@ def test_pyarrow_filesystem_option(tmp_path, fs):
     assert_eq(ddf, df)
 
 
+@PYARROW_MARK
+@pytest.mark.slow
+def test_pyarrow_filesystem_option_real_data():
```

Comment:
Does this test need to talk to S3? IOW, should it have the `network` marker?

Reply:
Ah, yeah. It probably makes sense to add a `@pytest.mark.network` decorator. I just pushed https://github.com/dask/dask/pull/10653 with this change. Let me know if this is what you had in mind.